main.nf

```diff
@@ -3391,6 +3391,21 @@ process svdb_merge {
 			manta = []
 			tiddit = []
 			gatk = []
+
+			/*
+			 Order in which VCFs are merged matters when the merged SV
+			 is annotated with final position/length, which affects
+			 artefact matching in loqusdb.
+
+			 A possibly better way to sort here would be to sort the
+			 file by familial-relation (e.g. always sort proband-mother-father)
+			 this would ensure the same merge-order regardless of sample-id
+			 */
+
+			mantaV = mantaV.collect { it.toString() }.sort()
```

Comment:
I would prefer not to overwrite the same variable with new content. This can (and often has for me) yield subtle bugs where it is hard to know what information the variable contains.